src/g_editor.c

```diff
@@ -11,10 +11,29 @@
 #include "g_undo.h"
 #include "s_utf8.h" /*-- moo --*/
 #include <string.h>
+#include <math.h>
+#include <stdlib.h>
+#include "g_kbdnav.h"
 #ifdef _MSC_VER  /* This is only for Microsoft's compiler, not cygwin, e.g. */
 #define snprintf _snprintf
 #endif
 
+typedef struct _editor_private {
+#ifdef HAVE_KEYBOARDNAV
+    struct _kbdnav kbdnav; /*keyboard navigation*/
```

Comment:
since you don't directly access the members of kbdnav, this could be `struct _kbdnav * kbdnav`. This would help encapsulation because you don't expose the internals of your structure.

Reply:
:+1: I agree with @Spacechild1

Reply:
fixed in b768c3b1b916e3dce649a30f3ca9209d630fdc18